samples/openthread/cli/snippets/l2/snippet.yml

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+name: l2
+append:
+  EXTRA_CONF_FILE: l2.conf
```

Comment:
Why do we need to create a dedicated snippets just to pass single argument? Can't we just add EXTRA_CONF_FILE=l2.conf in the sample.yml? I think it would be more reasonable, at least if we would have different .overlay files for different boards and we need to pull it.